src/scripts/main.js

```diff
@@ -1,3 +1,221 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+const theads = [...table.tHead.rows[0].cells];
+const tRows = [...table.tBodies[0].rows];
+
+function setUpSorting() {
+  for (const thead of theads) {
+    thead.addEventListener('click', sortByColumn);
+    thead.setAttribute('asc', true);
```

Comment:
Consider using `thead.setAttribute('asc', 'true');` for consistency, as attribute values are typically strings.